content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md

```diff
@@ -114,9 +98,9 @@ Import specific class methods from the module:
 from influxdb_client_3 import InfluxDBClient3, Point, WriteOptions
 ```
 
-- [`influxdb_client_3.InfluxDBClient3`](#class-influxdbclient3): a class for interacting with InfluxDB
-- [`influxdb_client_3.Point`](#class-point): a class for constructing a time series data
-  point
+- [`influxdb_client_3.InfluxDBClient3`](#class-influxdbclient3): a class for interacting with InfluxDB.
```

Comment:
Thanks for catching the punctuation.

Just FYI, we'd typically do the reverse--we don't use periods if all the bullets are phrases (incomplete sentences). If at least one needs a complete sentence, then we'll end them all with periods to be consistent.

I'm not going to nitpick, though, as long as it looks consistent.